### ISSUE TITLE
What the structure should to have file application.properties? 

### ISSUE THREAD

**Issue by EgorKuts** [wcrbrm/java-kucoin-api#3]
Help me, pls. When i trying to place order i have this exceptions: exchanges.kucoin.datatype.api.KucoinApiException: ERROR: UNAUTH, Login required . I think that occur due to  missing file application.properties .

**Comment by wcrbrm** [comment]
@EgorKuts 

`application.properties` example is:
KUCOIN_API_KEY=XXXXXXX
KUCOIN_SECRET=2323929239

**Comment by wcrbrm** [comment]
but there is also often `UNAUTH, Invalid nonce` which means their API is down

**Comment by EgorKuts** [comment]
Thank you. Have you any information about fix of Kucoin API? =)

**Comment by wcrbrm** [comment]
no =)  but this repositories is still under development, it requires a lot of things to be done

**Comment by EgorKuts** [comment]
I have fixed a little bit your code(Creating of sign, adding of some headers etc). Now it works. Unfortunately this repository was't available. https://github.com/EgorKuts/java-kucoin-api

**Comment by wcrbrm** [comment]
yes. thanks for the input. I see how you are trying to solve "nonce" issue.

However, as it is connected to the timestamp of their particular server
my recommendation for you is to make first request (to any address, including the one that doesn't exist) and keep timestamp difference between their server and your client.

This is what will be prepared in production version of this library. And we're just seeking for thread-safe solution on this ;)

**Comment by josephblan** [comment]
I develop a project on android and study your library.

I have a huge problems with "POST" requests when trying to Create Order.
Any POST request that contains of two or more attributes (parameters e.g. amount=10&price=1.1&type=BUY) always returns HTTP_BAD_REQUEST.
POST with one parameter like Change Language or [Set Default Currency](https://kucoinapidocs.docs.apiary.io/#reference/0/currencies-plugin/set-default-currency) works well.

I'm interested in your experience of using APIs from KuCoin. Do APIs work well for you?

**Comment by EgorKuts** [comment]
Hi. I have solved problems with placing orders on Kucoin API in this repository: https://github.com/EgorKuts/java-kucoin-api . You can see at tests. It's working well14.03.2018, 17:58, "josephblan" <notifications@github.com>:I develop a project on android and study your library.
I have a huge problems with "POST" requests when trying to Create Order.
Any POST request that contains of two or more attributes (parameters e.g. amount=10&price=1.1&type=BUY) always returns HTTP_BAD_REQUEST.
POST with one parameter like Change Language or Set Default Currency works well.
I'm interested in your experience of using APIs from KuCoin. Do APIs work well for you?
